string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,9 @@
+/* exported isVowel */
+function isVowel(char) {
+  var vowel = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'];
+  for (var i = 0; i < vowel.length; i++) {
+    if (char === vowel[i]) {
+      return true;
+    }
+  } return false;
```

Comment:
We haven't covered it in the class, but there's also something called a `switch` statement, that you can use to handle this! More info [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) if you were curious about reading

```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```